stable/drone/templates/deployment-server.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/secrets: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
+        checksum/secrets: {{ include (print $.Template.BasePath "/secret-shared-rpc.yaml") . | sha256sum }}
```

Comment:
Wouldn't be also an option to merge the secrets in a single file? I wonder if having multiple secrets is really required. There is also no checksum on the new secret.